treetime/vcf_utils.py

```diff
@@ -494,7 +576,7 @@ def handleDeletions(i, pi, pos, ref, delete, pattern):
         #Write it out - Increment positions by 1 so it's in VCF numbering
         #If no longer variable, and explained, don't write it out
         if printPos:
-            output = ["MTB_anc", str(pos), ".", refb, ",".join(uniques), ".", "PASS", ".", "GT"] + calls
+            output = [chrom_name, str(pos), ".", refb, ",".join(uniques), ".", "PASS", ".", "GT"] + calls
```

Comment:
Running through `bcftools view` errors with
```
[W::vcf_parse] contig '1' is not defined in the header. (Quick workaround: index the file with tabix.)
[vcf.c:1265 bcf_write] Unchecked error (1), exiting.
```

This error's not new, but I should fix it here